lms/envs/common.py

```diff
@@ -2129,7 +2129,7 @@
     'bulk_enroll',
 
     # Student Identity Verification
-    'lms.djangoapps.verify_student',
+    'lms.djangoapps.verify_student.apps.VerifyStudentConfig',
```

Comment:
Unsure how others feel about this convention. But I'd rather keep the `INSTALLED_APPS` name simply `lms.djangoapps.verify_student` and specify a `default_app_config` in `__init__.py`. I think specifying the AppConfig name should be reserved for the case where a single Django app has more than one AppConfig, as the Django docs show. It's just easier to read the `INSTALLED_APPS` list that way.